blockchain/types/accountkey/account_key.go

```diff
@@ -63,7 +63,7 @@ type AccountKey interface {
 	Equal(AccountKey) bool
 
 	// Validate returns true if the given public keys are verifiable with the AccountKey.
```

Comment:
Is there no need to explain the int type return value?

Reply:
@hyochan-brown Thanks for the review! The return value description was written at the account_key_weighted_multi_sig.go.. Uhm, however the int return value is no longer necessary because i changed the implementation not to return int.